README.md

```diff
@@ -49,7 +54,8 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+town[:residents].delete( "Belle" )
+town[:castle][:guests] = "Belle"
```

Comment:
On this second line you're setting the `guests` key to this string "Belle". Make sure to use array syntax to add "Belle" to the empty `guests` array.